src/components/ui/Input/InputUsageExample.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * This file is used as an example for the Dialog component.
+ * Once you're done with the example, you can delete this file.
+ */
+
+import { Input, NumberInput, SearchInput } from '..';
```

Comment:
Using a relative path is not a good practice except `./`. You could use `./` for this case.